pom.xml

```diff
@@ -193,4 +193,15 @@
 			</plugins>
 		</pluginManagement>
 	</build>
+	<profiles>
```

Comment:
Yes.  Currently with Java8 if we generate the javadoc compilation will fail.  With problems like return & param types not documented.  Some of these problems we can (should) fix as they are hand written classes, but the majority are from the generated classes.  So there is little we can do.